### ISSUE TITLE
Marquee effect for file names

### ISSUE THREAD

**Issue by shuvashish76** [MuntashirAkon/AppManager#1169]
### Please check before submitting an issue

- [X] I am using the latest version of App Manager
- [X] I have searched the issues and haven't found anything relevant
- [X] I have read the docs

### Describe a description of the new feature

AM > FM
Marquee effect to view full names. Useful incase the beginning & end of names are same for multiple files.This should be opt-in feature because some users (from other FMs) find fully displayed names visually annoying.

![am_fm](https://github.com/MuntashirAkon/AppManager/assets/48226312/6c562b58-1b83-47bd-9402-363c7bc5578a)
Currently it's "Ellipsize the middle".

### Describe the solution you'd like

![mf_gif](https://github.com/MuntashirAkon/AppManager/assets/48226312/5652c5a9-98ff-4a8e-a0d3-1d91c94d684d)
"Marquee" effect from [MF](https://github.com/zhanghai/MaterialFiles).

**Comment by MuntashirAkon** [comment]
> This should be opt-in feature because some users (from other FMs) find fully displayed names visually annoying.

I agree. Ellipsis in the middle of the filenames is very much the standard implementation.

**Comment by MuntashirAkon** [comment]
Implemented via bdb01d325ef94400aa220ea8bb3c3cbba1177ff6 and f0dc38c1be979b8ae8e0c2838be6e11d6373b32f
